### PR TITLE
Support configuring HTTP port via ENV variable

### PR DESCRIPTION
**Pull request by deric**
Allow changing default HTTP port.

### CODE REVIEW

File: gatherer/src/main/java/de/zalando/pgobserver/gatherer/GathererApp.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public static void main(final String[] args) {
 
         try {
             LOG.info("Starting restlet server");
-            new Server(Protocol.HTTP, 8182, GathererApp.class).start();
+            new Server(Protocol.HTTP, Integer.parseInt(getEnv("PORT0", "8182")), GathererApp.class).start();
```
Comment (slitsche) [inline gatherer/src/main/java/de/zalando/pgobserver/gatherer/GathererApp.java]: Could we have a more descriptive name for that environment variable? Maybe "HTTP_PORT"?

Comment (deric) [reply gatherer/src/main/java/de/zalando/pgobserver/gatherer/GathererApp.java]: Variable `PORT0` is automatically set by Marathon orchestrator, thus allowing seamless integration.

**Comment by deric** [comment]
Sure, I've updated the README to include that variable.

**Comment by slitsche** [comment]
👍

**Comment by hjacobs** [comment]
@deric thanks!

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**